tox.ini

```diff
@@ -9,17 +9,7 @@ deps = keyring
        django
        webtest
        nose
+       coverage>=3.6,<3.99
        unittest2
 setenv = PYTHONPATH=../google_appengine
-
-[testenv:py26]
-commands = nosetests --ignore-files=test_oauth2client_appengine\.py
```

Comment:
Commit 51852247883c780b68006d6d6b4888b6efbdf6c7 deleted all of the oauth2client tests since they were included in oauth2client.